aws-alb-ingress-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,128 @@
+kind: Deployment
+metadata:
+  name: {{ include "aws-alb-ingress-controller.fullname" $root }}
+  labels: {{ include "aws-alb-ingress-controller.labels" $root | nindent 4 }}
+    {{- range $key, $value :=.Values.deployment.labels }}
+    {{ $key }}: {{ tpl $value $root | quote }}
+    {{- end }}
+  annotations:
+  {{- range $key, $value := .Values.deployment.annotations }}
+    {{ $key }}: {{ tpl $value $root | quote }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.deployment.replicaCount }}
+  selector:
+    matchLabels: {{ include "aws-alb-ingress-controller.instance-labels" $root | nindent 6 }}
+  template:
+    metadata:
+      labels: {{ include "aws-alb-ingress-controller.instance-labels" $root | nindent 8 }}
+        {{- range $key, $value := .Values.podLabels }}
+        {{ $key }}: {{ tpl $value $root | quote }}
+        {{- end }}
+      annotations:
+        {{- range $key, $value := .Values.podAnnotations }}
+        {{ $key }}: {{ tpl $value $root | quote }}
+        {{- end }}
+    spec:
+      {{- if .Values.priorityClassName }}
+      priorityClassName: "{{ .Values.priorityClassName }}"
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args:
+        - --cluster-name={{ required "specify clusterName via --set clusterName=YourClusterName" .Values.clusterName }}
+        {{- with .Values.scope.ingressClass }}
+        - --ingress-class={{ . }}
+        {{- end }}
+        {{- with .Values.scope.singleNamespace }}
+        - --watch-namespace={{ default .Release.Namespace .Values.scope.watchNamespace }}
+        {{- end }}
+        {{- if not .Values.autoDiscoverAwsRegion }}
+        - --aws-region={{ .Values.awsRegion }}
+        {{- end }}
+        {{- if not .Values.autoDiscoverAwsVpcID }}
+        - --aws-vpc-id={{ .Values.awsVpcID }}
+        {{- end }}
+        {{- with .Values.extraArgs }}
+{{ toYaml . | indent 8 }}
+        {{- end }}
+        env:
+        {{- range $key, $value := .Values.extraEnv }}
+          - name: {{ $key }}
+            value: "{{ $value }}"
+        {{- end }}
+        {{- range $key, $value := .Values.envValueFromConfigMap }}
+          - name: {{ $key }}
+            valueFrom:
+              configMapKeyRef:
+                name: {{ $value.name }}
+                key: {{ $value.key }}
+        {{- end }}
+        {{- range $key, $value := .Values.envValueFromEnvSecret }}
+          - name: {{ $key }}
+            valueFrom:
+              secretKeyRef:
+                name: {{ $value.name }}
+                key: {{ $value.key }}
+        {{- end }}
+        {{- with .Values.envFromConfigmap }}
+          envFrom:
+            - configMapRef:
+                name: {{ . }}
+        {{- end }}
+        {{- with .Values.envFromSecret }}
+          envFrom:
+            - secretRef:
+                name: {{ . }}
+        {{- end }}
```

Comment:
@cw-sakamoto 

Wouldn't the following code be simpler?

```
envFrom: {{ toYaml .Values.extraEnvFrom | nindent  12 }}
```

Reply:
There is a duplicate `envFrom` in `envFromConfigmap` and `envFromConfigmap `.

Reply:
Oh...you are right...I fixed
https://github.com/chatwork/charts/pull/147/commits/eb126d9931d12312d340e168fc8d85a0cc1f9ddc